inc/agent.class.php

```diff
@@ -488,12 +487,12 @@ public function prepareInputForUpdate($input) {
 
       // send wipe to the agent
       if (isset($input['wipe']) && $input['wipe'] != '0') {
-         $input['wipe'] == '1';
```

Comment:
What ? I wrote that ?

Reply:
yes you did and it has been there for a lot of time